src/test/java/core/basesyntax/service/FruitServiceImplTest.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.fruitrecord.FruitRecord;
+import core.basesyntax.service.activities.ActivityHandler;
+import core.basesyntax.service.activities.BalanceHandler;
+import core.basesyntax.service.activities.PurchaseHandler;
+import core.basesyntax.service.activities.ReturnHandler;
+import core.basesyntax.service.activities.SupplyHandler;
+import core.basesyntax.service.activities.TypeOfActivities;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
+    private static final FruitRecord recordOrangeBalance =
+            new FruitRecord(new String[]{"b", "orange", "100"});
+    private static final FruitRecord recordOrangeReturn =
+            new FruitRecord(new String[]{"r", "orange", "8"});
+    private static final FruitRecord recordOrangeSupply =
+            new FruitRecord(new String[]{"s", "orange", "10"});
+    private static final FruitRecord recordOrangePurchase =
+            new FruitRecord(new String[]{"p", "orange", "100"});
+    private static final FruitRecord recordCherryBalance =
+            new FruitRecord(new String[]{"b", "cherry", "8"});
+    private static final FruitRecord recordCherryReturn =
+            new FruitRecord(new String[]{"r", "cherry", "26"});
+    private static final FruitRecord recordCherrySupply =
+            new FruitRecord(new String[]{"s", "cherry", "1011"});
+    private static final FruitRecord recordCherryPurchase =
+            new FruitRecord(new String[]{"p", "cherry", "1000"});
```

Comment:
I don't think it should be as constants, lets create them in `setUp()` method and add to `List<FruitRecord> records`